microsoft-edge/devtools-guide-chromium/whats-new/2022/06/devtools-103.md

```diff
@@ -0,0 +1,158 @@
+*  Improved reliability of **Quick View** controls.
+*  Improved behavior of opening DevTools by using keyboard shortcuts.
+*  Fixed an issue with navigating to a specific line of code in the **Sources** tool by using keyboard shortcuts.
+*  Restored the `Ctrl`+`Shift`+`F` shortcut to open **Search** in **Quick View**:<!-- todo: macOS keys? -->
+
+![The Search keyboard shortcut opens the Search tool in Quick View.](devtools-103-images/focus-mode-search-shortcut.png)
+
+See also:
+* [Simplify DevTools using Focus Mode](../../../experimental-features/focus-mode.md)
+
+
+<!-- ====================================================================== -->
+## The 3D View tool responds better in narrow layouts
+
+<!-- Title: 3D View: Now with more responsive design -->
+<!-- Subtitle: The 3D View toolbar now wraps around in a narrower window. -->
+
+As folks use the **3D View** tool more, especially with the addition of Composited Layers (formerly known as **Layers**), we have found opportunities to improve the UI based on your feedback.  We've made the layout of the 3D View toolbar more responsive so that your development isn't hindered even when DevTools is narrow.  Keep the feedback coming as you check out 3D View!
+
+![Responsive layout of toolbar in 3D View tool.](devtools-103-images/3d-view-responsive-layout.png)
+
+See also:
+* [Navigate webpage layers, z-index, and DOM using the 3D View tool](../../../3d-view/index.md)
+
+
+<!-- ====================================================================== -->
+## Fix: The Close button for DevTools is now accessible in high contrast mode
+
+<!-- Title: Fix: In high contrast mode, the Close button for DevTools is now accessible -->
+<!-- Subtitle: In previous versions of Microsoft Edge, the Close button wasn't visible in high contrast mode, but this issue has been fixed in Microsoft Edge 103. -->
+
+In previous versions of Microsoft Edge, the **Close** button for DevTools didn't render correctly in high contrast mode.  In Microsoft Edge 103, this issue has been fixed.  The **Close** (![Close icon.](../../../media/close-devtools-icon-light-theme.png)) button for DevTools is displayed correctly in high contrast mode:
+
+![The Close DevTools (X) button in high contrast mode.](devtools-103-images/close-button-high-contrast.png)
+
+See also:
+* [Navigate DevTools with assistive technology](../../../accessibility/navigation.md)
+* [Windows high contrast mode](/fluent-ui/web-components/design-system/high-contrast)
+
+
+<!-- ====================================================================== -->
+## Redesigned Welcome tool
+
+<!-- Title: Welcome tool has expanded content and videos -->
+<!-- Subtitle: The redesigned Welcome tool has expanded Learn and What's New sections, including a new video series. -->
+
+In Microsoft Edge 103, the **Welcome** tool has a new design with improved, expanded content.  It's easier than ever to find content and to contact the DevTools team.
```

Comment:
```suggestion
In Microsoft Edge 103, the **Welcome** tool has a new design with improved, expanded content.  It's easier than ever to find documentation, videos, and information on new features and bug fixes.
```

We don't have a link to the GitHub repo so I don't think it makes sense to say it's easier to contact the DevTools team. Reeza, we should follow up on adding a GitHub repo link to the Welcome tool